client/components/segmented-control/style.scss

```diff
@@ -80,31 +80,31 @@
 	.segmented-control__item {
 		&.is-selected {
 			.segmented-control__link {
-				border-color: var( --color-accent );
-				background-color: var( --color-accent );
+				border-color: var( --color-primary );
+				background-color: var( --color-primary );
 				color: $white;
 
 				&:focus {
-					background-color: lighten( $blue-medium, 2.5% );
+					background-color: var( --color-primary-light );
 				}
 			}
 
 			+ .segmented-control__item .segmented-control__link {
-				border-left-color: var( --color-accent );
+				border-left-color: var( --color-primary );
 			}
 		}
 	}
 
-	.segmented-control__link:focus {
-		background-color: lighten( $blue-light, 22.5% );
-	}
+	// .segmented-control__link:focus {
+	// 	background-color: lighten( $blue-light, 22.5% );
```

Comment:
I'm not sure about a replacement for these color and would love some feedback @drw158 on what color would work?

Reply:
The equivalent color of `lighten( $blue-light, 22.5% )` is roughly `blue-0`, but it's so desaturated, I think we can use `var( --color-neutral-0` here.